amt/site/templates/projects/new.html.j2

```diff
@@ -34,12 +34,16 @@
                     </div>
                     <h2 class="utrecht-heading-2 rvo-heading--no-margins">{% trans %}AI Act Profile{% endtrans %}</h2>
                     <p class="rvo-text rvo-text--no-margins">
-                        {% trans %}The AI Act profile provides insight into, among other things, the type of AI system and the associated obligations from the European AI Act.{% endtrans %}
-                        <a class="rvo-link rvo-link--normal"
-                           href="https://ai-act-decisiontree.apps.digilab.network/">{% trans %}The AI Act decision tree {% endtrans %}</a>
-                        {% trans %}helps you determine the value of these fields.{% endtrans %}
+                        {% trans %}The AI Act profile provides insight into, among other things, the type of AI system and the associated obligations from the European AI Act. If you already know the type of AI system you can fill in the below fields. Otherwise, you can find your AI Act Profile with the AI Act Decision tree.{% endtrans %}
+
                     </p>
                     <div class="utrecht-form-fieldset rvo-form-fieldset">
+                    <p class="utrecht-button-group">
```

Comment:
```suggestion

```

Reply:
Is the paragraph really needed if you just have 1 button?

Reply:
It is needed for right alignment.